### DISCUSSION TITLE
is it right `process()` method is deprecated?

### DISCUSSION THREAD

**Discussion by SeonJK** [euphony-io/euphony#140] [Q&A] [answered]
Hi! I forked this repository and cloned into my local. And I have a question.

In `EuTxManager.java`, `process()` method  is deprecated and replaced by `setCode()` method.
However, in `README.md`, `GETTING_STARTED.md` and `GETTING_STARTED_KR.md`, they still describe `process()` method for generating sound with count.

My question is...
- surely `process()` is deprecated, do we have to update documents from `process()` to `setCode()`?
- If we have to update documents, I think it is necessary to have another method for generating sound with count. How about update `process()` to `play()` method?

**Comment by designe** [comment, marked as answer]
Hi @SeonJK 

You're right! you found comment & documentation error
https://github.com/euphony-io/euphony/blob/f463bfcc01c0dc1a05ecfe745eb166d55b38f1d5/euphony/src/main/java/co/euphony/tx/EuTxManager.java#L143

Actually, The `setCode()` is not a replacement for the `process()`. We need to modify here (`setCode()` -> `play()`)

https://github.com/euphony-io/euphony/blob/f463bfcc01c0dc1a05ecfe745eb166d55b38f1d5/euphony/src/main/java/co/euphony/tx/EuTxManager.java#L146

As you mentioned, `play()` is the proper function for generating sound. so, we need to update comment & documentation.
Thanks for finding them! Could you make a PR for this?

**Reply by SeonJK** [reply]
Thanks for reply!!
I will make a PR for this.

**Reply by SeonJK** [reply]
I just made a PR!!

**Comment by SeonJK** [comment]
I have one more question here.
in `GETTING_STARTED.md`, `process(-1)` generates sound infinitely.
does `play(-1)` work same as `process(-1)`?

**Reply by designe** [reply]
https://github.com/euphony-io/euphony/blob/f463bfcc01c0dc1a05ecfe745eb166d55b38f1d5/euphony/src/main/java/co/euphony/tx/EuTxManager.java#L152

Actually, process() function is wrapping of play function. exactly same :)
